docs/markdown/Fs-module.md

```diff
@@ -8,6 +8,8 @@ available starting with version 0.53.0.
 Non-absolute paths are looked up relative to the directory where the
 current `meson.build` file is.
 
+If specified, `~` is expanded to the user home directory.
```

Comment:
This should probably be "a leading ~ is expanded"?

Reply:
Yes